roles/provision-metrics-apb/tasks/provision-app-metrics.yml

```diff
@@ -0,0 +1,115 @@
+- name: Postgresql PVC
+  k8s_v1_persistent_volume_claim:
+    name: postgresql-data
+    namespace: "{{ namespace }}"
+    labels:
+      app: app-metrics-api
+      service: prometheus
+    access_modes:
+      - ReadWriteMany
+    resources_requests:
+      storage: "10Gi"
```

Comment:
Flagging this as it may be too restrictive for trying out this service on shared clusters.
Can we expose that as APB parameter?

Reply:
Yup there are other things to be variablised as I clean this up like the postgres and api server env variables